modules/serverless-kafka-sasl-source.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-sasl-source_{context}"]
+= Configuring SASL authentication for Kafka sources
+
+_Simple Authentication and Security Layer_ (SASL) is used by Apache Kafka for authentication. If you use SASL authentication on your cluster, users must provide credentials to Knative for communicating with the Kafka cluster, otherwise events cannot be produced or consumed.
+
+.Prerequisites
+
+* You have cluster or dedicated administrator permissions on {product-title}.
```

Comment:
```suggestion
* You have a cluster or dedicated administrator permissions on {product-title}.
```

Reply:
I don't think that this makes sense, it's not referring to "a cluster" but "cluster admin" or "dedicated admin" permissions